README.md

```diff
@@ -0,0 +1,73 @@
+# Installation
+
+To reproduce the website in your local machine follow these steps.
+
+## Clone repo and submodules
+
+    git clone --recurse-submodules git@github.com:PyLadiesBerlin/website.git
+
+For git < 2.11 use:
+
+    git clone --recursive git@github.com:PyLadiesBerlin/website.git
+
+## Update submodules
+
+    git submodule update
+
+## Setup an environment
+
+### Using python
+
+    python3 -m venv .env  # use python if it is a python > 3
+    source .env/bin/activate
+
+#### Install requirements
+
+    pip install -r requirements.txt
+
+### Using conda or mamba
+
+With conda:
+
+    conda create -n pyladies
+    conda install -c conda-forge pelican markdown beautifulsoup4
```

Comment:
shouldn't it use the requirements.txt to avoid needed to edit the readme for changes in requirements? (I have no idea how conda and mamba works)

Reply:
it doesn't work with reqs txt because it's pulling from a diff place than pypi